lualib/lua_redis.lua

```diff
@@ -843,7 +843,7 @@ local function add_redis_script(script, redis_params)
 end
 exports.add_redis_script = add_redis_script
 
-local function exec_redis_script(id, params, callback, args)
+local function exec_redis_script(id, params, callback, args, aargs)
```

Comment:
Why `aargs` instead of `keys` and `args` as it is used in Redis `eval` command? In fact, even `args` was a misguiding name.

Reply:
`args` is keys and `aargs` are argument-arguments. :) we can rename them...